### PR TITLE
DOCS-2136: Update README.md some touchups

### PR DESCRIPTION
**Pull request by sguequierre**
fixing some typos and old link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,16 +28,18 @@ Depending on the type of models configured, the module implements:
...
-### Example of config with a camera and transform camera
-The following json config file includes the following ressources:
-- TorchvVision module
-- modular ressource (Torchvision vision service)
--  a camera
-- a transform camera 
+The following JSON config file includes the following resources:
+- TorchVision module
+- modular resource (TorchVision vision service)
+- a webcam [camera](https://docs.viam.com/components/camera/)
```
Comment (JessamyT) [inline README.md]: What is the objective of including this link? Should it be this?
```suggestion
- a [webcam camera](https://docs.viam.com/components/camera/webcam/)
```

**Review by JessamyT — APPROVED** [review]
LGTM % question

File: README.md (hunk 1 above)
Comment (sguequierre) [reply README.md]: originally it was just camera here (although configuration is webcam) so I thought having the link to just camera might be helpful bc of the camera api. this is also fine though and more standardized!

Comment (JessamyT) [reply README.md]: okay whatever you think makes the most sense--you have more context than I do

**Review by Rob1in — APPROVED** [review]
thanks!

**Comment by npentrel** [comment]
@Rob1in can you admin-merge over the test failure? That seems unrelated

**Comment by Rob1in** [comment]
@npentrel done, ty! will take a look at the test

**Pull request merged**